GameServer/Logic/Maps/SummonersRift.cs

```diff
@@ -15,7 +15,7 @@ namespace LeagueSandbox.GameServer.Logic.Maps
 {
     class SummonersRift : Map
     {
-        private List<Vector2> _blueTopWaypoints = new List<Vector2>
+        private List<Vector2> BLUE_TOP_WAYPOINTS = new List<Vector2>
```

Comment:
writing it in uppercase doesn't make it a constant, add "readonly".
` private readonly List<Vector2> BLUE_TOP_WAYPOINTS = new List<Vector2>`